tests/framework/Optimizers/GeneticAlgorithms/tests

```diff
@@ -57,90 +57,153 @@
+    input = 'discrete/unconstrained/testGAMaxwoRepDifferentDist.xml'
     [./data]
       type = OrderedCSV
-      output = 'MaxwoReplacementDiffDist/opt_export_0.csv'
+      output = 'discrete/unconstrained/MaxwoReplacementDiffDist/opt_export_0.csv'
       rel_err = 1e-3
       zero_threshold = 1e-5
     [../]
   [../]
 
   [./MaxwoReplacementConvAHDp]
     type = 'RavenFramework'
-    input = 'testGAMaxwoRepConvAHDp.xml'
+    input = 'discrete/unconstrained/testGAMaxwoRepConvAHDp.xml'
     [./data]
       type = OrderedCSV
-      output = 'MaxwoReplacementConvAHDp/opt_export_0.csv'
+      output = 'discrete/unconstrained/MaxwoReplacementConvAHDp/opt_export_0.csv'
       rel_err = 1e-3
       zero_threshold = 1e-5
     [../]
   [../]
 
-  [./GA_discreteIntWithReplacement]
+  [./discreteIntWithReplacement]
     type = 'RavenFramework'
-    input = 'GA_discreteIntWithReplacement.xml'
+    input = 'discrete/unconstrained/GA_discreteIntWithReplacement.xml'
     [./csv]
      type = OrderedCSV
-     output = 'discreteIntWithReplacement/PrintOptOut_export.csv discreteIntWithReplacement/PrintOptOut.csv'
+     output = 'discrete/unconstrained/discreteIntWithReplacement/PrintOptOut_export.csv ./discrete/unconstrained/discreteIntWithReplacement/PrintOptOut.csv'
      rel_err = 0.001
     [../]
   [../]
 
   [./GA_knapsackBase]
     type = 'RavenFramework'
-    input = 'GA_knapsackBase.xml'
+    input = 'discrete/unconstrained/GA_knapsackBase.xml'
     [./csv]
      type = OrderedCSV
-     output = 'simpleKnapsack/PrintOptOut_export.csv  simpleKnapsack/PrintOptOut.csv'
+     output = 'discrete/unconstrained/simpleKnapsack/PrintOptOut_export.csv  ./discrete/unconstrained/simpleKnapsack/PrintOptOut.csv'
      rel_err = 0.001
     [../]
   [../]
 
   [./GA_knapsackBaseUnifXover]
     type = 'RavenFramework'
-    input = 'GA_knapsackBaseUnifXover.xml'
+    input = 'discrete/unconstrained/GA_knapsackBaseUnifXover.xml'
+    [./csv]
+     type = OrderedCSV
+     output = 'discrete/unconstrained/simpleKnapsackUnifXover/PrintOptOut_export.csv  ./discrete/unconstrained/simpleKnapsackUnifXover/PrintOptOut.csv'
+     rel_err = 0.001
+    [../]
+  [../]
+
+  [./MaxwReplacementSphereConstrained]
+    type = 'RavenFramework'
+    input = 'discrete/constrained/testGAMaxwRepSphereConstrained.xml'
+    heavy = true
+    max_time = 2000
+    [./csv]
+     type = OrderedCSV
+     output = 'discrete/constrained/MaxwReplacementSphereConstrained/opt_export_0.csv'
+     rel_err = 0.001
+    [../]
+  [../]
+
+[./MinwReplacementConstrained]
+    type = 'RavenFramework'
+    input = 'discrete/constrained/testGAMinwRepConstrained.xml'
+    heavy = true
+    max_time = 2000
     [./csv]
      type = OrderedCSV
-     output = 'simpleKnapsackUnifXover/PrintOptOut_export.csv  simpleKnapsackUnifXover/PrintOptOut.csv'
+     output = 'discrete/constrained/MinwReplacementConstrained/opt_export_0.csv'
      rel_err = 0.001
     [../]
   [../]
 
+  [./mishraBirdConstrained]
```

Comment:
A lot of gold files have been removed, but I do not see that you add the new gold files. Please check all the tests that you modified, and try to add all related gold files.